### DISCUSSION TITLE
对新功能的想法

### DISCUSSION THREAD

**Discussion by YuShai** [ant-design/ant-design#48150] [Ideas]
希望新功能可以支持一个选择器，下拉选择列表里可以选择星期和时分秒，选择内容是一个下拉列表内展示，例如选择星期几后选择时分秒

**Comment by zombieJ** [comment]
不太明白，有图不？看看长什么样子

**Reply by YuShai** [reply]
![1711618059587](https://github.com/ant-design/ant-design/assets/44886448/0bbe03ee-2049-467f-901f-7681c6d84544)

类似与这个样式

**Reply by YuShai** [reply]
不是类似，就是这样的功能
